src/main/java/org/apache/sysds/runtime/controlprogram/federated/FederatedWorkerHandler.java

```diff
@@ -115,6 +131,77 @@ private FederatedResponse constructResponse(FederatedRequest request) {
 		}
 	}
 
+	private FederatedResponse createFrameEncodeMeta(FederatedRequest request) {
```

Comment:
Im not really a fan of this method located inside the workerHandler.

Reply:
Do you mean you don't like what the function does, or you would prefer if we but it in a different class?
If the first is the problem, maybe some details (or I refer to the comment further above), otherwise I believe we should anyway start thinking about splitting `FederatedWorkerHandler` up into some/one class that handles some types of tasks. Thoughts?

Reply:
It is purely a comment on location.
I think adding a FedEncoder file inside `sysds/runtim/transform/encode/fedencode.java`, could be a good because it keeps the functionality collected at the same place the other transform operations are located.

`anyway start thinking about splitting FederatedWorkerHandler` 
- I'm always a fan of splitting things apart :smile: (maybe to much)